package/upgrade/upgrade_node.sh

```diff
@@ -586,13 +589,24 @@ command_single_node_upgrade() {
   # Add logging related kube-audit policy file
   patch_logging_event_audit
 
+  echo "wait for fleet bundles before upgrading RKE2"
+  # wait all fleet bundles in limited time
+  wait_for_fleet_bundles
```

Comment:
prefer to check the bundle status in the [upgrade_manifests.sh ](https://github.com/harvester/harvester/blob/master/package/upgrade/upgrade_manifests.sh) as well.